yt_dlp/extractor/tiktok.py

```diff
@@ -219,7 +220,15 @@ def extract_addr(addr, add_meta={}):
                 'source_preference': -2 if 'aweme/v1' in url else -1,  # Downloads from API might get blocked
                 **add_meta, **parsed_meta,
                 'format_note': join_nonempty(
-                    add_meta.get('format_note'), '(API)' if 'aweme/v1' in url else None, delim=' ')
+                    add_meta.get('format_note'), '(API)' if 'aweme/v1' in url else None, delim=' '),
+                **({  # fixup slideshow formats
+                    'format_note': 'Music track',
+                    'ext': 'mp3',
+                    'acodec': 'mp3',
+                    'vcodec': 'none',
+                    'width': None,
+                    'height': None,
+                } if determine_ext(url) == 'mp3' else {})
```

Comment:
Looks ugly inline imo. Maybe it should be a function

Reply:
agreed, function it is